src/main/java/controller/RacingGameController.java

```diff
@@ -0,0 +1,27 @@
+package controller;
+
+import java.util.List;
+import model.Car;
+import model.RacingGame;
+import view.InputView;
+import view.ResultView;
+
+public class RacingGameController {
+    // 게임 실행 및 결과 출력
+    public void startRacingGame() {
+
+        List<Car> carNames = InputView.getCarNames();
+        int raceRounds = InputView.getRaceRounds();
+
+        RacingGame racingGame = new RacingGame(carNames);
+        ResultView.printRaceResult();
+
+        for (int i = 0; i < raceRounds; i++) {
+            racingGame.race();
+            ResultView.printRaceStatus(racingGame.getCars());
+        }
+
+        List<Car> winners = racingGame.getWinners();
+        ResultView.printWinners(winners);

```

Comment:
```suggestion
        ResultView.printWinners(racingGame.getWinners());
```

어느 쪽이 더 낫다고 생각하시나요? 지후의 의견이 궁금합니다!

Reply:
가독성을 위해 변수 할당 방식을 사용했는데,
`racingGame.getWinners()` 메서드명이 이미 직관적이라
리뷰어님이 말씀해주신 방향으로 리팩토링하는 게 더 나을 것 같다고 판단해서
리펙토링 진행했습니다 감사합니다 !